macrel/ORFs_prediction.py

```diff
@@ -0,0 +1,54 @@
+def create_pyrodigal_orffinder():
+    # generating orf_finder
+    import pyrodigal
+    gorf = pyrodigal.OrfFinder(closed=True,
+                               min_gene=33,
+                               max_overlap=0)       
+    morf_finder = pyrodigal.OrfFinder(meta=True,
+                                      closed=True,
+                                      min_gene=33,
+                                      max_overlap=0)
+    return gorf, morf_finder
+    
+
+def ppyrodigal_out(contig, ind, idx, pred):
+    orfid = f'{contig}_{ind}'
+    seconid = f'ID={idx}_{ind}'
+    part = ''.join([str(int(pred.partial_begin)),
+                    str(int(pred.partial_end))])
+    part = f'partial={part}'
+    st = f'start_type={pred.start_type}'
+    motif = f'rbs_motif={pred.rbs_motif}'
+    sp = f'rbs_spacer={pred.rbs_spacer}'
+    gc = f'gc_cont={round(pred.gc_cont, 3)}'
+    last = ';'.join([seconid, part, st, motif, sp, gc])
+    header = f'>{orfid} # {pred.begin} # {pred.end} # {pred.strand} # {last}'
+    return (header, pred.translate())
+
+
+def predict_genes(infile, ofile):
+    import random
+    import pandas as pd
+    from .fasta import fasta_iter
+    random.seed(1991)
+    predictions = []
```

Comment:
Generally speaking, I'm not a big fan of this type of structure: why not directly write to the output (particularly, if using `atomicwrites`)?

Reply:
implemented as asked